client/settings/settings-manager/index.js

```diff
@@ -140,6 +140,36 @@ const SettingsManager = () => {
 	const [ isTransactionInputsValid, setTransactionInputsValid ] = useState(
 		true
 	);
+	const { isLoading } = useSettings();
+
+	// Simulate scroll to hash on React component.
+	useEffect( () => {
+		if ( isLoading ) {
+			return;
+		}
+
+		const { hash } = window.location;
+
+		if ( ! hash ) {
+			return;
+		}
+
+		const element = document.querySelector( hash );
+
+		if ( ! element ) {
+			return;
+		}
+
+		const headerOffset = 110; // header size + margin
```

Comment:
This works right now, but getting the header height might be safer since it's always possible it could change, right?

Reply:
Fixed on https://github.com/Automattic/woocommerce-payments/pull/5848/commits/d45b9af6219858b40aa0324300769efa848a72be.